.gitignore

```diff
@@ -29,6 +29,7 @@ local/
 # editor files
 .idea/
 .*.sw[op]
+/*.iml
```

Comment:
Unrelated change, should be removed from this commit.

Reply:
I see other editor files being ignored as well, why not?

Reply:
Should be a separate pull request.